MobileSDK/sdk/proguard-rules.pro

```diff
@@ -15,3 +15,6 @@
 #-keepclassmembers class fqcn.of.javascript.interface.for.webview {
 #   public *;
 #}
+
+# Keep
+-keep class org.lantern.mobilesdk.**
```

Comment:
This is the fix.  I put it in the MobileSDK project and then reference it from the lantern-mobile project.